### DISCUSSION TITLE
undefined: Error: listen EADDRINUSE: address already in use :::6415

### DISCUSSION THREAD

**Discussion by rlarudqls630** [mKeRix/room-assistant#1089] [Q&A]
Hi
I am trying to set up room-assistant trough docker.
However, I was not able to finish the set up due to this error

undefined: Error: listen EADDRINUSE: address already in use :::6415

can anyone help? 
Thank you

**Comment by mKeRix** [comment]
This error just means that something else is already using the network port 6415 on your machine. Maybe you already have another instance of room-assistant running?

**Reply by pjdonovan84** [reply]
how do you run multiple instances of RA if running one blocks the other? I have 4 pis, but can't seem to get more than 2 to run at the same time without that error

**Reply by pjdonovan84** [reply]
Answering my own question.  After playing around with it - I noticed if you unplug all the pi's or turn off each service, then start ONE, it will start working and in my BLE configuration, it shows each device and it's RSSI value.  Keep that one up - the others can be brought up, but each of those will give the error message and not display the devices, but the original Pi will!  Keep that open while you start up the rest and you'll see they are all working!
